src/docs/asciidoc/hazelcast_clients.adoc

```diff
@@ -986,8 +986,8 @@ Hazelcast Cloud is disabled for the Java client, by default (`enabled` attribute
 See this Hazelcast Cloud link:https://hazelcast.com/products/cloud/[web page^]
 for more information on Hazelcast Cloud.
 
-NOTE: Since this is a REST based discovery, you need to enable the REST listener service,
-too, by setting the `hazelcast.rest.enabled` property to `true`.
+NOTE: Since this is a REST based discovery, you need to enable the REST listener service.
```

Comment:
@Serdaro I actually haven't found the code that uses some REST endpoint on IMDG from this cloud configuration strategy so I gave a pretty vague description here to turn on the REST endpoints. Maybe someone from the cloud team can give better instructions.